tools/setup_helpers/ninja_builder.py

```diff
@@ -65,7 +65,7 @@ def patch(obj, attr_name, val):
         if self.compiler.compiler_type == 'msvc':
             import distutils.msvccompiler
             import distutils.msvc9compiler
-            if sys.version[0] == 2:
+            if sys.version[0] == '2':
```

Comment:
This change looks dodgy. Not because it's wrong (`sys.version` is indeed a string) but that we should have used `verison_info` in `platform` instead.

Reply:
@ezyang Done!